day03/deploy_django_app.sh

```
@@ -40,6 +40,8 @@ required_restarts() {
 
 # Function to deploy the Django app
 deploy() {
+    echo "switching to project directory"
+    cd django-notes-app
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)